src/AcceptanceTests/NServiceBus.Transport.AzureServiceBus.AcceptanceTests.csproj

```diff
@@ -2,6 +2,8 @@
 
   <PropertyGroup>
     <TargetFrameworks>net472;net6.0</TargetFrameworks>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\NServiceBus.snk</AssemblyOriginatorKeyFile>
```

Comment:
this should use the NServiceBusTests.snk otherwise you'll get into hard to trace problems around type scanning